json_logging/README.md

```diff
@@ -46,45 +60,53 @@ def handle_event(event, context):
         aws_request_id=context.aws_request_id,
         function_name=context.function_name,
         function_version=context.function_version,
+        invoked_function_arn=context.invoked_function_arn,
         log_stream_name=context.log_stream_name,
     )
     logger.info(f"Starting {__name__}", extra={"event": event})
 ```
 
-### Additional info
+### "Library" code
+
+Outside the main module where you configure the logger, the pattern is:
```

Comment:
Is this the current correct pattern ? Or incorrect? @tvogels01

Reply:
Trick question :) 
It's the correct pattern. It's not the current pattern in all of our code.